plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -224,6 +224,9 @@ func (p *Plugin) admitPodCreate(nodeName string, a admission.Attributes) error {
 	if len(pod.OwnerReferences) > 1 {
 		return admission.NewForbidden(a, fmt.Errorf("node %q can only create pods with a single owner reference set to itself", nodeName))
 	}
+	if len(pod.OwnerReferences) == 0 {
```

Comment:
nit: else if

Reply:
The previous if block has return statement. So is this if block.

It seems else is not needed.